repositories.yaml

```diff
@@ -123,3 +123,19 @@ firefox-reality:
   dependencies:
     - org.mozilla.components:service-glean
     - org.mozilla.components:support-sync-telemetry
+lockwise-ios:
```

Comment:
Will you be tackling Android as a separate PR? (https://bugzilla.mozilla.org/show_bug.cgi?id=1600613)

Reply:
Sure, we can add Android while we're at it